rca/project_styleguide/templates/patterns/atoms/sprites/link-anchor/link-anchor.html

```diff
@@ -0,0 +1,4 @@
+<a class="link link--anchor" herf="{{value.href}}">
+    {{ value.text }}
+    <svg><use xlink:href="#arrow"></use></svg>
```

Comment:
Think we should be adding height and widths to SVG tags?

Reply:
I've set them via the stylesheet although I think either way is fine.